packages/vue-app/template/App.js

```diff
@@ -79,7 +84,24 @@ export default {
     'nuxt.err': 'errorChanged'
   },
   <% } %>
+  computed: {
+    isOffline() {
+      return !this.isOnline
+    }
```

Comment:
If they can be used from everywhere through `this.$nuxt.isOffline` and `this.$nuxt.isOnline`, I suggest adding these new type definitions of the `$nuxt` object :
```ts
isOffline: boolean;
isOnline: boolean;
```
here https://github.com/nuxt/nuxt.js/blob/dev/packages/vue-app/types/vue.d.ts#L29

As this PR is related to the `vue-app` package, you can just add them in a new commit I think :)

I'm not sure if they should be placed before or after `$loading` though

Reply:
@Atinux look here ^

Reply:
Added, thanks @kevinmarrec 

I added them before since $loading is added on `mounted()` hook and also is not available if `loading: false` is defined in `nuxt.config.js`